tests/dummy/app/templates/application.hbs

```diff
@@ -1,4 +1,4 @@
-{{osf-navbar loginAction=(action 'login')}}
+{{new-osf-navbar loginAction=(action 'login') currentService='PREPRINTS'}}
```

Comment:
@sheriefvt is working on a helper that reads from the config file in https://openscience.atlassian.net/browse/EOSF-553